server/pulp/server/content/web/views.py

```diff
@@ -143,23 +147,30 @@ def get(self, request):
         if request.environ['wsgi.url_scheme'] != 'http':
             if not allow_access(request.environ, host):
                 # Not Authorized
+                logger.debug(_('Denying {host} access to {path}.').format(host=host, path=path))
```

Comment:
If possible, it might be good to say why the access is denied (e.g. because of ACLs, or whatever). Also, this might be better at loglevel INFO.